hcl2template/testdata/build/basic.pkr.hcl

```diff
@@ -1,66 +1,138 @@
 
 // starts resources to provision them.
 build {
-    from "src.amazon-ebs.ubuntu-1604" {
-        ami_name = "that-ubuntu-1.0"
-    }
-
-    from "src.virtualbox-iso.ubuntu-1204" {
-        // build name is defaulted from the label "src.virtualbox-iso.ubuntu-1204"
-        outout_dir = "path/"
-    }
+    from_sources = [
+        "src.amazon-ebs.ubuntu-1604",
+        "src.virtualbox-iso.ubuntu-1204",
+    ]
```

Comment:
I simplified this; before I planned to allow to overload a source config but this will require refactoring a few things ( like splitting Validation from Decoding ( Configuration ).